pagerduty/user.go

```diff
@@ -402,6 +455,13 @@ func (s *UserService) CreateNotificationRule(userID string, rule *NotificationRu
 	return v.NotificationRule, resp, nil
 }
 
+func isSameNotification(existingRule, newRule *NotificationRule) bool {
```

Comment:
Maybe add a comment saying that there might be a legitimate conflict

Reply:
It should be fine as we are comparing the contact method ID?